js/repeater.js

```diff
@@ -105,6 +106,10 @@
 				pageIncrement: null
 			});
 		});
+		this.$search.on('canceled.fu.search', function (e, value) {
+			self.$element.trigger('canceled.fu.repeater', value);
```

Comment:
As discussed in the sprint retrospective, this also needs to re-render the repeater (see the handler above for how to call render). Keep in mind that render will ignore "re-renders" if its in the middle of a render, so the render method itself will need to be updated to allow stopping and re-rendering a render that it mid-render (which is the case here during a cancelled search).